logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/UserSessionScope.kt

```diff
@@ -1735,8 +1742,10 @@ class UserSessionScope internal constructor(
             this,
             userScopedLogger,
         )
-    val messages: MessageScope
-        get() = MessageScope(
+    }
+
+    val messages: MessageScope by lazy {
```

Comment:
We noticed while checking in the profiler,  that we were creating many instances of `MessageScope`.